easybuild/easyconfigs/b/bsddb3/bsddb3-6.2.6-fosscuda-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'PythonPackage'
+
+name = 'bsddb3'
+version = '6.2.6'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://pypi.org/project/bsddb3/'
+description = """bsddb3 is a nearly complete Python binding of the
+Oracle/Sleepycat C API for the Database Environment, Database, Cursor,
+Log Cursor, Sequence and Transaction objects."""
+
+toolchain = {'name': 'fosscuda', 'version': '2018b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['42d621f4037425afcb16b67d5600c4556271a071a9a7f7f2c2b1ba65bc582d05']
+
+osdependencies = [('libdb-dev')]
+
+dependencies = [('Python', '2.7.15')]
+
+use_pip = True
+download_dep_fail = True
+
+preinstallopts = 'unset LIBS && '
```

Comment:
@akesandgren Why is this required exactly? Maybe add a comment above this line to explain?